docs/setup.md

```diff
@@ -1,44 +1,61 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Installation
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Continuous Delivery**. 
+2. From the list of widgets that appears, search for and select **CICD Configuration Wizard**. 
 
-3. Click the card of the **Continuous Delivery** solution pack card.
+3. Click the card of the **CICD Configuration Wizard** widget card.
 
-4. Click **Configure** on the bottom to begin configuration.
+4. Click **Install** on the bottom to begin installation.
 
```

Comment:
Add a note, that this widget is only useful when using with CICD SP.
@cs-abhishek-shukla

Reply:
It is in `readme.md`.
>The **CICD Configuration** widget helps set up source control and streamline the content development lifecycle across multiple FortiSOAR environments designated as Development, Staging, and Production through a configuration wizard.

Reply:
Added in setup.md